ansible/roles/hardening_partitions/defaults/main.yml

```diff
@@ -0,0 +1,36 @@
+---
+root_fs: ext4
+swap_size: 1g
```

Comment:
is it enough for swap? do we set swapiness somewhere?

Reply:
can be increased using a separate variable. There is information in the readme https://github.com/Aparavi-Operations/aparavi-infrastructure/pull/6/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R53